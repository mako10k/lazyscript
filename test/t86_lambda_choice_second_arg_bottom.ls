# 左腕が選ばれた後の2引数目でパターンマッチ失敗 → ⊥（右腕は未評価）
(
  (\(Some ~x) (Some ~y) -> ~x)
|
  (\(Some ~a) (Some ~b) -> 999)
) (Some 1) (None);
