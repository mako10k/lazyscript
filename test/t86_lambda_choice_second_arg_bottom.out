E: <bottom msg="lambda match failure" at <unknown>:1.1: >
